EddystoneBeacon.cpp

```diff
@@ -76,6 +76,33 @@ void EddystoneBeacon::begin(char power, const char* uri) {
 
   this->_bleCharacteristic.broadcast();
 }
+/***Following is the code for broadcasting TLM ****/
+/***Implementation in the examples folder***/
+uint8_t buffer[14]= {
+  0x20,
+  0x00,
+  0x00,
+  0x00,
+  0x00,
+  0x80,
+  0x00,
+  0x00,
+  0x00,
+  0x00,
+  0x00,
+  0x00,
+  0x00,
+  0x00
+}; //global
+unsigned char len = 14; //global
+void EddystoneBeacon::tlm_begin(){
```

Comment:
Maybe we don't even need a buffer, just splitting vars for temperature, battery level, adv count, boot time etc.